samples/zip-split-rabbit-binder/README.adoc

```diff
@@ -0,0 +1,59 @@
+= Split unzipped content into Rabbit Binder destination
+
+This sample demonstrates how out of the box from catalog and custom functions can be composed, and how the final result can be bound to RabbitMQ destination by the https://spring.io/projects/spring-cloud-function[Spring Cloud Stream] framework.
+The goal of this sample is to poll zip files, unzip them, and emit messages to RabbitMQ for every line of text from those zip entries.
+
+The sample uses these dependencies:
+
+[source,groovy]
+----
+implementation 'org.springframework.cloud:spring-cloud-stream-binder-rabbit'
+implementation 'org.springframework.integration:spring-integration-zip'
+implementation 'org.springframework.cloud.fn:spring-file-supplier'
+implementation 'org.springframework.cloud.fn:spring-splitter-function'
+----
+
+The first one is for binding output of the composed function into a RabbitMQ destination.
+The second one is for `UnZipTransformer`, which we use for a custom function to unzip polled files by the `fileSupplier`.
+The `splitterFunction` is used in a `FileSplitter` mode to read lines from unzipped entries and emit each of them as an individual message.
+Essentially, we are splitting twice: zip entries, and content of each file.
+
+The composition is like this: `fileSupplier|unzipFunction|splitterFunction|flattenFunction`.
+(The `flattenFunction` will be explained latter).
+This becomes a `Supplier<Flux<Mesage<?>>>` and we bind it into a RabbitMQ `unzipped_data_exchange` using Spring Cloud Stream.
```

Comment:
Re-iterating that "this" is the "output of the above composition" may be helpful. 

> The result of this composition is a `Supplier<Flux<Mesage<?>>>` which we then bind into a RabbitMQ `unzipped_data_exchange` using Spring Cloud Stream.